packages/cli/test/functional/test_site/expected/siteData.json

```diff
@@ -5,7 +5,12 @@
       "src": "bugs/index.md",
       "title": "Open Bugs",
       "headings": {},
-      "headingKeywords": {}
+      "headingKeywords": {},
+      "frontmatter": {
+        "title": "Open Bugs",
+        "globalOverrideProperty": "Overridden by global override",
+        "globalAndFrontMatterOverrideProperty": "Overridden by global override"
+      }
```

Comment:
Let's add in a [functional test](https://markbind.org/devdocs/devGuide/workflow.html#updating-functional-tests) into any one of the test pages for this to generate a page with `"frontmatter": { "keywords": ... }"`.

Reply:
Alright! I will add a functional test in my next commits!